crunch-core/src/main/java/org/apache/crunch/io/impl/FileTargetImpl.java

```diff
@@ -178,7 +206,7 @@ public void handleOutputs(Configuration conf, Path workingPath, int index) throw
       if (useDistributedCopy) {
         LOG.info("Source and destination are in different file systems, performing distributed copy from {} to {}", srcPattern,
             path);
-        handeOutputsDistributedCopy(conf, srcPattern, srcFs, dstFs, maxDistributedCopyTasks);
+        handleOutputsDistributedCopy(dstFsConf, srcPattern, srcFs, dstFs, maxDistributedCopyTasks);
```

Comment:
This is a cherry-pick merge mistake causing the build to fail.  I'll fix in a second and make sure the build and all tests are passing.   This was originally developed on an internal fork and reviewed with my colleagues, @noslowerdna and @mkwhitacre.